### DISCUSSION TITLE
SEO: how to ensure admin ui is not indexed in google?

### DISCUSSION THREAD

**Discussion by pnodet** [keystonejs/keystone#8985] [Questions] [answered]
Our admin ui is on a subdomain but I want to make sure a random people won't find it in google

**Comment by unrevised6419** [comment, marked as answer]
Hey, @pnodet! The simple way is to use [`extendExpressApp`](https://keystonejs.com/docs/config/config#extend-express-app) and use `express.static` middleware

First, create `public` folder in your project, and add a `robots.txt` file in there. You should have `public/robots.txt` with below content:

**public/robots.txt**

```
User-agent: *
Disallow: /
```

Then add `express.static` middleware to server everything from that folder.

**keystone.ts**

```ts
// ...
import express from 'express';

export default withAuth(
  config({
    // ...
    server: {
      extendExpressApp(app) {
        app.use(express.static('public'))
        // ^ this line serves everything from `public` folder
      },
    }
  })
);
```

The other way is to configure [local storage](https://keystonejs.com/docs/guides/images-and-files#storing-assets-in-local), but it's much tricky

**Reply by pnodet** [reply]
Crystal clear
